libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -233,6 +233,32 @@ void GCS_MAVLINK::send_power_status(void)
                                   hal.analogin->power_status_flags());
 }
 
+void GCS_MAVLINK::send_battery_status(const AP_BattMonitor &battery, const uint8_t instance) const
+{
+    uint16_t voltages[MAVLINK_MSG_BATTERY_STATUS_FIELD_VOLTAGES_LEN];
+    memset(&voltages, 0xff, sizeof(uint16_t)*MAVLINK_MSG_BATTERY_STATUS_FIELD_VOLTAGES_LEN); // fill with UINT16_MAX for unknown cells
```

Comment:
Because memset fills memory with a constant byte, so passing anything larger can be misleading. I think 0xFF is still the correct thing to do here, although I agree that (uint8_t)UINT16_MAX will produce the same result, I find that limiting the input expression is usually an easier approach.